dat/missions/trader/anxiousmerchant.lua

```diff
@@ -130,3 +131,8 @@ function tick()
    end
    misn.osdCreate(_("Help the Merchant"), osd_msg)
 end
+
+function abort ()
```

Comment:
I would not add a faction hit penalty for aborting the mission. Currently, the Trader Guild is also somewhat removed from the game (it is set as unknown to the player) pending a complete revamp and rethinking of them.

Reply:
Fair enough, I'll fix that... is it something that would be reasonable after or as part of the revamp?

I just feel like aborting the mission kinda dumps the merchant in the doodoo from an in-universe perspective and that this is made clear before accepting the mission... it's also a relatively small change (even if it is greater than the late arrival one - on purpose!)

Reply:
It would be reasonable, but since the player expects most missions to be abortable without consequence I think it would need something like:
1. Maybe a yes/no prompt to confirm with a message indicating faction hit
2. Probably not have the standing go below 0

Reply:
Both good points I'll keep in mind :)